catroid/src/androidTest/java/org/catrobat/catroid/test/io/filepicker/ListProjectFilesTest.java

```diff
@@ -48,23 +51,31 @@
 	private File tmpFolder;
 
 	@Before
-	public void setUp() {
+	public void setUp() throws IOException {
 		tmpFolder = new File(InstrumentationRegistry.getTargetContext().getCacheDir(), "ListProjectFilesTestTmp");
 		if (tmpFolder.exists()) {
```

Comment:
good catch below! **BUT** you should be aware that you should 
`if (tmpFolder.isDirectory()) { ... }` instead of `if (tmpFolder.exists()) { ... }`
if you want to be sure it is a directory you're trying to delete (deleting a file with `StorageOperations.deleteDir(` is going to cause an IOException), if the exception is what you want you should probably think about changing it some way that it becomes more obvious that it is what you want.